src/Microsoft.TestPlatform.Extensions.BlameDataCollector/XmlReaderWriter.cs

```diff
@@ -55,7 +55,7 @@ protected XmlReaderWriter(IFileHelper fileHelper)
         /// The file Path.
         /// </param>
         /// <returns>File path</returns>
-        public string WriteTestSequence(List<TestCase> testSequence, string filePath)
```

Comment:
>TestCase [](start = 45, length = 8)

check for outcome/result

Reply:
There is no outcome field in TestCase object.